index.html

```diff
@@ -232,6 +232,27 @@ <h4 class="connect-link padding">LET'S CONNECT</h4>
 
       </section>
     </section>
+    <section class="contact-form-section">
+      <h2 class="contact-me-title">Contact me</h2>
+      <p class="contact-description">If you have an application you are interested in developing,
+        a feature that you need built or a project that needs
+        coding. I’d love to help with it</p>
```

Comment:
- [ ] Excellent work implementing this section of the form. You still have some slight tweaking to do before we can close this 😄. The design text is centered but yours is not. Please make this fix and get the all clear

**Design**
![scrn1](https://user-images.githubusercontent.com/99656687/220343127-347da08b-1249-44ae-bb6f-52aca76d1ac4.png)
**Your implementation**
![scrn2](https://user-images.githubusercontent.com/99656687/220343189-ebc88ea7-0e9b-477c-865b-c5969d54b3d3.png)